.gitlab-ci.yml

```diff
@@ -14,6 +14,38 @@ stages:
   - build
   - release_resources
 
+# Whether and how to update uberenv
+.run_update_uberenv: &run_update_uberenv |
+  [[ -n "${UPDATE_UBERENV}" ]] && ./scripts/gitlab/update-uberenv.sh "${UPDATE_UBERENV}"
+
+####
+# Template
+.srun_build_script:
+  script:
+    - *run_update_uberenv
+    #Use pre-existing allocation if any
+    - JOBID=$(squeue -h --name=${PROJECT_ALLOC_NAME} --format=%A)
+    - ASSIGN_ID=$(if [[ -n "${JOBID}" ]]; then echo "--jobid=${JOBID}"; fi)
+    - EXEC_PREFIX="srun ${ASSIGN_ID}"
+    #DEPENDENCIES
+    - RESOURCES="-t 20 -N 1 -n 1 -c 4"
```

Comment:
The intel compiler took 22 minutes last time i ran it... might need to bump that up a bit.

Reply:
Yes, I just updated this branch, but haven’t merged it yet to my latest changes regarding allocation time. WIP.